ngraph/test/type_prop/dft.cpp

```diff
@@ -0,0 +1,308 @@
+
+        EXPECT_EQ(dft->get_element_type(), element::f32);
+        ASSERT_TRUE(dft->get_output_partial_shape(0).same_scheme(s.ref_output_shape));
+    }
+}
+
+TEST(type_prop, dft_constant_axes_and_there_are_no_signal_size_dynamic_shapes2)
+{
+    struct ShapesAndValues
+    {
+        PartialShape input_shape;
+        Shape axes_shape;
+        PartialShape ref_output_shape;
+    };
+
+    std::vector<ShapesAndValues> shapes_and_values = {
+        {{2, 180, 180, Dimension(1, 18)}, {2}, {2, 180, 180, Dimension(1, 18)}},
+        {{2, 180, Dimension(7, 500), 2}, {2}, {2, 180, Dimension(7, 500), 2}},
+        {{2, 180, Dimension(7, 500), Dimension(1, 18)},
+         {2},
+         {2, 180, Dimension(7, 500), Dimension(1, 18)}},
+        {{2, Dimension(7, 500), 180, 2}, {2}, {2, Dimension(7, 500), 180, 2}},
+        {{2, Dimension(7, 500), 180, Dimension(1, 18)},
+         {2},
+         {2, Dimension(7, 500), 180, Dimension(1, 18)}},
+        {{2, Dimension(7, 500), Dimension(7, 500), 2},
+         {2},
+         {2, Dimension(7, 500), Dimension(7, 500), 2}},
+        {{2, Dimension(7, 500), Dimension(7, 500), Dimension(1, 18)},
+         {2},
+         {2, Dimension(7, 500), Dimension(7, 500), Dimension(1, 18)}},
+        {{Dimension(0, 2), 180, 180, 2}, {2}, {Dimension(0, 2), 180, 180, 2}},
+        {{Dimension(0, 2), 180, 180, Dimension(1, 18)},
+         {2},
+         {Dimension(0, 2), 180, 180, Dimension(1, 18)}},
+        {{Dimension(0, 2), 180, Dimension(7, 500), 2},
+         {2},
+         {Dimension(0, 2), 180, Dimension(7, 500), 2}},
+        {{Dimension(0, 2), 180, Dimension(7, 500), Dimension(1, 18)},
+         {2},
+         {Dimension(0, 2), 180, Dimension(7, 500), Dimension(1, 18)}},
+        {{Dimension(0, 2), Dimension(7, 500), 180, 2},
+         {2},
+         {Dimension(0, 2), Dimension(7, 500), 180, 2}},
+        {{Dimension(0, 2), Dimension(7, 500), 180, Dimension(1, 18)},
+         {2},
+         {Dimension(0, 2), Dimension(7, 500), 180, Dimension(1, 18)}},
+        {{Dimension(0, 2), Dimension(7, 500), Dimension(7, 500), 2},
+         {2},
+         {Dimension(0, 2), Dimension(7, 500), Dimension(7, 500), 2}},
+        {{Dimension(0, 2), Dimension(7, 500), Dimension(7, 500), Dimension(1, 18)},
+         {2},
+         {Dimension(0, 2), Dimension(7, 500), Dimension(7, 500), Dimension(1, 18)}}};
+
+    for (const auto& s : shapes_and_values)
+    {
+        auto data = std::make_shared<op::Parameter>(element::f32, s.input_shape);
+        auto axes_input = std::make_shared<op::Parameter>(element::i64, s.axes_shape);
+        auto dft = std::make_shared<op::v7::DFT>(data, axes_input);
+
+        EXPECT_EQ(dft->get_element_type(), element::f32);
+        ASSERT_TRUE(dft->get_output_partial_shape(0).same_scheme(s.ref_output_shape));
+    }
+}
+
+TEST(type_prop, dft_constant_axes_and_there_are_signal_size_dynamic_shapes2)
+{
+    struct ShapesAndValues
```

Comment:
The same picture as before.

Reply:
All `type_prop` tests for DFT were rewritten as parametrized tests.